src/menu.asm

```diff
@@ -1677,71 +2114,55 @@ execute_numfield_sound:
 
 execute_numfield_word:
 {
-    ; grab the memory address (long)
-    LDA [!DP_CurrentMenu] : INC !DP_CurrentMenu : INC !DP_CurrentMenu : STA !DP_Address
-    LDA [!DP_CurrentMenu] : INC !DP_CurrentMenu : STA !DP_Address+2
-
-    ; grab minimum and maximum values
-    LDA [!DP_CurrentMenu] : INC !DP_CurrentMenu : INC !DP_CurrentMenu : STA !DP_Minimum
-    LDA [!DP_CurrentMenu] : INC !DP_CurrentMenu : INC !DP_CurrentMenu : INC : STA !DP_Maximum ; INC for convenience
+    ; check for A, B, Y, Left, or Right
+    LDA !IH_CONTROLLER_PRI_NEW : BIT #$43C0 : BEQ .done
```

Comment:
Is this necessary? Should probably remove it. It's not allowing X for some reason.